.changeset/cold-foxes-applaud.md

```diff
@@ -0,0 +1,11 @@
+---
+'@finos/legend-extension-mapping-generation': major
+'@finos/legend-graph': minor
+'@finos/legend-studio': minor
+---
+
+WHAT Add model loader extension for relational mapping generation, no breaking changes as the
+extension is not added to studio yet
```

Comment:
Lets have 2 changeset files. 

One with `legend-graph` and `legend-studio` (patch) with message `Add extension mechanism for loading models with model loader.

Another one with `legend-extension-mapping-generation` (minor) with message `Add model loader extension for relational mapping generation to allow users to merge their M2M mapping chain into
one relational mapping`